include/wx/qt/datectrl.h

```diff
@@ -0,0 +1,64 @@
+///////////////////////////////////////////////////////////////////////////////
+// Name:        wx/qt/datectrl.h
+// Purpose:     wxDatePickerCtrl for Qt
+// Author:      Ali Kettab
+// Modified by:
```

Comment:
Not worth updating the PR just for this, but, just FYI, we don't use this line any more in the new files, it doesn't seem worth it with `git shortlog` providing the same information much more reliably.